core/src/main/java/org/apache/calcite/sql/SqlDropQuark.java

```diff
@@ -27,9 +27,9 @@
  * A <code>SqlDropQuark</code> is a node of a parse tree which represents a DROP DDL
  * statements for Quark.
  */
-public class SqlDropQuark extends SqlCall {
-  public static final SqlSpecialOperator OPERATOR =
-      new SqlSpecialOperator("DROP_QUARK", SqlKind.OTHER_DDL);
+public abstract class SqlDropQuark extends SqlCall {
+  public static SqlSpecialOperator operator;
```

Comment:
Don't make this static. Different objects who are inherited by this abstract class can have different operator. static would make sure all the objects share the same value which is wrong.